docs/sources/reference/run.md

```diff
@@ -583,6 +584,28 @@ This means processes in container can be executed on cpu 1 and cpu 3.
 
 This means processes in container can be executed on cpu 0, cpu 1 and cpu 2.
 
+### Blkio constraint
```

Comment:
A better structure for this is:  https://gist.github.com/fb433f62445d80cacf48.git

In tech docs, you typically don't want to talk about features that "might" be coming. Creates an anticipation in the reader that may or may not ever be resolved.  A feature either is or is not available.
--->

### Block IO bandwidth (Blkio) constraint

By default, all containers get the same proportion of block IO bandwidth
(blkio). This proportion is 500. To modify this proportion, change the
container's blkio weight relative to the weighting of all other running
containers using the `--blkio-weight` flag.

The `--blkio-weight` flag can set the weighting to a value between 10 to 1000.
For example, the commands below create two containers with different blkio
weight:

    $ sudo docker run -ti --name c1 --blkio-weight 300 ubuntu:14.04 /bin/bash
    $ sudo docker run -ti --name c2 --blkio-weight 600 ubuntu:14.04 /bin/bash

If you do block IO in the two containers at the same time, by, for example:

time dd if=/mnt/zerofile of=test.out bs=1M count=1024 oflag=direct

You'll find that the proportion of time is the same as the proportion of blkio
weights of the two containers.

>**Note**: The blkio weight setting is only available for direct IO. Buffered IO
>is not currently supported.